DisCatSharp.Docs/articles/misc/hosting.md

```diff
@@ -22,6 +22,65 @@ Any modern hardware will work just fine, new or used.
 
 Depending on how complex your bot is, you may even consider purchasing a Raspberry Pi ($35).
 
+# Termux
+
+For those who don't have some PC or other hardware lying around alternatively you could use your phone. Using [Termux](https://termux.dev/en/) and a package called [proot-distro](https://github.com/termux/proot-distro), which we use to install a Debian vm and setup DotNET to run the bot. To those interested the steps are given below:
+
+## Requirements
+
+- An Android 5+
+- Termux
+- An internet connection
+
+## Setup
+
+- Initialize Termux
+```sh
+pkg update && pkg upgrade -y
+```
+*Note, it might ask you for input, just click enter and let the default option execute*
+
+- Install proot-distro package
+```sh
+pkg install proot-distro -y
+```
+
+- Install a Ubuntu vm
+```sh
+proot-distro install ubuntu -y
+```
+*Note, it may take a bit depending on your internet speed*
+
+- Login into Ubuntu and initialize Ubuntu
+```sh
+proot-distro login ubuntu
+```
+```sh
+apt-get update -y && apt-get upgrade -y
```

Comment:
From my knowledge, apt-get is supposed to be used for scripts and such. At the moment it doesn't really make a difference but to future proof: Use apt instead, especially because apt 3.0 actually makes the output a lot more usable when that's released.

Reply:
Alright, noted also cannot find a way to.update to apt 3.0, if you know how please provide information